WooCommerce/Classes/ViewRelated/Dashboard/JetpackInstall/JetpackInstallIntroView.swift

```diff
@@ -0,0 +1,116 @@
+import SwiftUI
+
+/// Hosting controller wrapper for `JetpackInstallIntroView`
+///
+final class JetpackInstallHostingController: UIHostingController<JetpackInstallIntroView> {
+    init(siteURL: String) {
+        super.init(rootView: JetpackInstallIntroView(siteURL: siteURL))
+    }
+
+    required dynamic init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setDismissAction(_ dismissAction: @escaping () -> Void) {
+        rootView.dismissAction = dismissAction
+    }
+}
+
+/// Displays the intro view for the Jetpack install flow.
+///
+struct JetpackInstallIntroView: View {
+    // Closure invoked when Close button is tapped
+    var dismissAction: () -> Void = {}
+
+    let siteURL: String
+
+    private var descriptionAttributedString: NSAttributedString {
+        let font: UIFont = .body
+        let boldFont: UIFont = font.bold
+        let siteName = siteURL.trimHTTPScheme()
+
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.alignment = .center
+
+        let attributedString = NSMutableAttributedString(
+            string: String(format: Localization.installDescription, siteName),
+            attributes: [.font: font,
+                         .foregroundColor: UIColor.text.withAlphaComponent(0.8),
+                         .paragraphStyle: paragraphStyle,
+                        ]
+        )
+        let boldSiteAddress = NSAttributedString(string: siteName, attributes: [.font: boldFont, .foregroundColor: UIColor.text])
+        attributedString.replaceFirstOccurrence(of: siteName, with: boldSiteAddress)
+        return attributedString
+    }
+
+    var body: some View {
+        VStack {
+            HStack {
+                Button(Localization.closeButton, action: dismissAction)
+                .buttonStyle(LinkButtonStyle())
+                .fixedSize(horizontal: true, vertical: true)
+                Spacer()
+            }
+
+            Spacer()
+
+            VStack(spacing: Constants.contentSpacing) {
+                Image(uiImage: .jetpackLogoImage)
+                    .resizable()
+                    .renderingMode(.template)
+                    .foregroundColor(Color(.jetpackGreen))
+                    .frame(width: Constants.jetpackLogoSize, height: Constants.jetpackLogoSize)
```

Comment:
nit: maybe we can let the logo auto-adjust its size with system font size via `@ScaledMetric`